__tests__/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`parameters query URI encoding should encode query parameters 1`] = `
+"curl --request GET \\\\
+     --url 'https://httpbin.org/anything?stringPound=somethign%26nothing%3Dtrue&stringHash=hash%23data&stringArray=where%5B4%5D%3D10&stringWeird=properties%5B%22%24email%22%5D%20%3D%3D%20%22testing%22&array=something%26nothing%3Dtrue&array=nothing%26something%3Dfalse&array=second%20item'"
+`;
+
+exports[`parameters query URI encoding should encode query parameters 2`] = `
+"const fetch = require('node-fetch');
+
+const url = 'https://httpbin.org/anything?stringPound=somethign%26nothing%3Dtrue&stringHash=hash%23data&stringArray=where%5B4%5D%3D10&stringWeird=properties%5B%22%24email%22%5D%20%3D%3D%20%22testing%22&array=something%26nothing%3Dtrue&array=nothing%26something%3Dfalse&array=second%20item';
+const options = {method: 'GET'};
+
+fetch(url, options)
+  .then(res => res.json())
+  .then(json => console.log(json))
+  .catch(err => console.error('error:' + err));"
+`;
+
+exports[`parameters query URI encoding should encode query parameters 3`] = `
+"const sdk = require('api')('https://example.com');
+
+sdk.get('/anything', {
+  stringPound: 'somethign%26nothing%3Dtrue',
+  stringHash: 'hash%23data',
+  stringArray: 'where%5B4%5D%3D10',
+  stringWeird: 'properties%5B%22%24email%22%5D%20%3D%3D%20%22testing%22',
+  array: 'something%26nothing%3Dtrue&array=nothing%26something%3Dfalse&array=second%20item'
```

Comment:
I need to add a test in `api` to make sure that this data gets sent as it should in both this and the `array: ['something%26nothing%3Dtrue', 'nothing%26something%3Dfalse', ...]` cases.